editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/CamelEditor.java

```diff
@@ -477,7 +477,10 @@ protected void pageChange(int newPageIndex) {
 						updateModelFromSource();
 						lastError = "";
 						if (newPageIndex == GLOBAL_CONF_INDEX) globalConfigEditor.reload();
-						if (newPageIndex == DESIGN_PAGE_INDEX) designEditor.refreshOutlineView();
+						if (newPageIndex == DESIGN_PAGE_INDEX) {
+							designEditor.switchContainer(); // needed to fix sync issue between props view and selected context element
```

Comment:
we need an automatic test to detect the issue.
Not sure it is easy with non-UI tests. maybe we can ask QE for that.
@lhein WDYT?